lib/src/widgets/selection/handle_overlay.dart

```diff
@@ -198,6 +198,7 @@ class _MathSelectionHandleOverlayState extends State<MathSelectionHandleOverlay>
                 context,
                 type,
                 widget.manager.preferredLineHeight,
+                null,
```

Comment:
To explain, this will break on `stable` because stable has a 3 positional parameter method.

Reply:
Makes sense!